File: proj/Logic/src/quoridor/logic/Node.java
package quoridor.logic;

public class Node {
	public int alpha, beta;
	public boolean maximizerNode;
	
	public int heuristicValue;
	
	public Node(boolean maxNode) {
		this.maximizerNode = maxNode;
		this.alpha = Integer.MIN_VALUE;
		this.beta = Integer.MAX_VALUE;
	}
	
	public Node(boolean maxNode, int pAlpha, int pBeta) {
		this.maximizerNode = maxNode;
		this.alpha = pAlpha;
		this.beta = pBeta;
	}
	
}

File: Assets/Scripts/GameAI/Node.cs
using UnityEngine;
using System.Collections;

public class Node
{
	public bool isMaximizer;
	public float alpha, beta;
	public float heuristicValue;

	public Node(bool isMaximizer)
	{
		this.isMaximizer = isMaximizer;
		this.alpha = float.MinValue;
		this.beta = float.MaxValue;
		this.heuristicValue = isMaximizer ? float.MinValue : float.MaxValue;
	}
}
